samples/wrapper/compat.h

```diff
@@ -0,0 +1,18 @@
+#ifndef COMPAT_H
```

Comment:
If you use already existing functions' implementations, this file is not needed.